### ISSUE TITLE
[CAIA A11y Research]: Ask VA VA.gov Team, Ask VA – Dashboard (September 2024)

### ISSUE THREAD

**Issue by tygindraux** [department-of-veterans-affairs/va.gov-team#92003]
> [!NOTE]
> **High-Level Overview**: ASK VA
> - **Research Timeframe**: September 17 - 24 `1 Slot Remains` _as of 9/19/2024 at 11:17 a.m. EST_
> - **Pilot 1**: Mobile with Screen Reader: `#92024` 
> - **Pilot 2:** Desktop with Alternative Nav.: `#92028`
> - **Assistive Tech Users**: 12 AT Users
>     - **3 cohorts with a min. of:** 3 screen reader; 2 alt. nav.; and 1 screen mag.
> - **Artifacts Available**: [Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Dashboard/Research%20plan.md)
> - **Testing Environment & Schedule:** [Staging](https://staging.va.gov/contact-us/ask-va-too/) & [Source of Truth](https://docs.google.com/spreadsheets/d/1ZFkfsq2LLKm6RIzclKqwyBrERCss_wPusm-7-aPKSWo/edit?gid=331828355#gid=331828355) _(also see table in this ticket)_
> - **Assistive Tech Being Used**: Screen Readers, Alternative Navigation and Screen Magnification.
> - **CAIA Internal Doc:** [**CAIA Raw Session Notes: Google Doc**](https://docs.google.com/document/d/1vuk0sTW6YV-4mnWf-an5DLlj0tgSD9nnuGa1Y8puVOM/edit?usp=sharing)
> - **Task Lists**: See task lists for next steps/definition of done

## CAIA Assignee(s)

<details><summary>Toggle to view information about the CAIA Team.</summary>

- **Pilot 1 (Mobile):** SK and Evan
- **Pilot 2 (Desktop):** Jamie, Sara and Evan
- **Research Session Support:**
    - Evan
    - Sara
    - Maria
    - Jamie
    - SK

</details>

## About the Team
- Team name: Ask VA
- OCTO-DE product owner: Becky Phung
- Product manager: CeeCee O'Connor
- Designer: Tyler Gindraux, Thomas Michaud
- FE engineer: Eddie Otero, Hemesh Patel
- Product/team Slack channel: #ask-va-public

<details><summary>Toggle to view more intake research ticket information</summary>

### Do you have an accessibility champion?

No, we don't have a champion (yet)

### Calendar checking

- [X] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

</details>

## Research Sessions

> [!TIP]
> - Providing CAIA A11ys with the time slots in advance for AT users, will allow a11ys to avoid conflicts on the CAIA A11y Capacity Calendar, and reach out to other a11ys, should CAIA not be available. 
> - Providing CAIA A11ys with the technology and devices the users will be testing with, will allow CAIA to prepare in advance for the sessions. 

<details><summary>Toggle to view research timing information.</summary>

### Timeframe

1. **Pilot Session Mobile Screen Reader**: `TBD` Around 9/6/2024
2. **Pilot Session Desktop Alt. Nav.**:`TBD` Around 9/6/2024

**Timeframe**: 
- September 17 - 24, 2024

### Length of sessions

- **Session length:**
    - 90 minutes (screen reader or alternative navigation tool)
    - 60 minutes (screen magnification tool)
- **Buffer time between sessions:** 30 minutes
- **Maximum sessions per day:** 2

</details>

###  Testing Schedule

CAIA A11Ys will be looking for device and AT information. We realize that some of this information will not be available until the day of the session. 

Please toggle and use the following table to enter the following information for each participant: Date and Time, Assistive tech (AT), Device and Operating System, Browser, and Skill level:

- **Dates**: Tuesday, Sept. 17 to Tuesday, Sept. 24, 2024
- **Times**: 9-10:30am, 11-12:30pm, 1-2:30pm, 3-4:30pm EST
- **Planning to Recruit 12 Veterans for**:
     - minimum of 3 sessions with screen reader users
     - minimum of 2 sessions with alternative navigation users
     - minimum of 1 session with a screen magnification user

#### Source of Truth
[Testing Schedule - Google Sheet](https://docs.google.com/spreadsheets/d/1ZFkfsq2LLKm6RIzclKqwyBrERCss_wPusm-7-aPKSWo/edit?gid=331828355#gid=331828355)  **_also see table in this ticket_**

<details><summary>Toggle to view the AT testing schedule</summary>

_Participant numbers changed on 9/12/2024. Changes and additional switches noted in bold for the participant column below, as of 9/13/2024 9:21 a.m. EST._

P# | Date and Time (EST) | AT | Device and OS | Browser | Skill | CAIA A11ys Available | STATUS
------------------|------------------|--------------|--------------|--------------|--------------|--------------|--------------
**PILOT MOBILE `92024`** | 9/11/2024 2-3 p.m. ET| VoiceOver, JAWS, etc `TBD` | Mobile, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Pilot | @SarahKay8 | ✅ **COMPLETED**
**PILOT DESKTOP `92028`** | 9/11/2024 11 a.m.  | Voice Command | Desktop | Chrome | Pilot | @coforma-jamie | ✅ **COMPLETED**
P1 | Tues. Sept. 17 9 a.m. | VoiceOver | **Mobile**; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice | @SarahKay8 | 🛑 **NO SHOW**
P2 | Tues. Sept. 17 11 a.m. | VoiceOver | **Mobile**; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Intermediate | @EvanAtCoforma | ✅ **COMPLETED**
P3 | Wed. Sept. 18 1 p.m. | VoiceOver (also uses Fusion) | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Intermediate | @EvanAtCoforma | ✅ **COMPLETED**
P4 | Wed. Sept. 18 3 p.m. | VoiceOver | **Desktop**; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Intermediate | @mutiaadhocteam | ✅ **COMPLETED**
P5 |Thurs. Sept. 19 1 p.m. | VoiceOver | **Mobile**; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Intermediate | @EvanAtCoforma | 🛑 **NO SHOW**
**P6** | Thurs. Sept. 19 3 p.m. | VoiceOver | **Mobile**; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Intermediate | @sara-amanda  | ✅ **COMPLETED**
P7 | Fri. Sept. 20 9 a.m. | Dragon | **Desktop**; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Advanced | @sara-amanda  | ⛔ **LIMITED TO NO USE OF DRAGON IN STUDY**
P8 | Fri. Sept. 20 11 a.m. | Dragon | **Desktop**; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Advanced | @sara-amanda | ⛔ **LIMITED TO NO USE OF DRAGON IN STUDY**
**P9** | Mon. Sept. 23 12 p.m. | Voice Control | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Advanced | @sara-amanda | ⛔ **Non-AT User**
**P10** | Mon. Sept. 23 3 p.m. | **Screen Magnification** - Enlarge screen on Safari using gestures | **Mobile** - Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Advanced | @SarahKay8 | 🛑 **NO SHOW**
**P11** | Tues. Sept. 24 9 a.m. | **Screen Magnification** - Windows 11 | **Desktop**; Windows | Chrome, etc. `TBD` | Advanced | @SarahKay8 | 🛑 **CANCELED**
**P12** | Mon. Sept. 23 10:30 a.m. | Text to talk Microsoft dictation | **Mobile**; - Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Advanced | **a11y TBD** | 🛑 **CANCELED**
**P13 - `NEW`** |Tues. 9/24/2024 at 9:30 a.m. | **Screen reader VoiceOver** | `Mobile - iPhone` | `TBD` | `Advanced` | @SarahKay8  | 🆕 **REPLACEMENT  for P11**
**P14 - `NEW`** |Tues. 9/24/2024 at noon | **Screen Maginification** | `NA`| `NA` | `NA | `NA` | 🆕  `No a11y Needed`
**P15 - `NEW`** | Tues. 9/24/2024 at 1 p.m.  | **ALT NAV - Dragon** | `Desktop` | `TBD` | `Advanced` | @sara-amanda (until 2:15 p.m. EST) | 🆕 **PENDING SESSION**
- **Consideration**: You can also consider asking participants more specific AT and device combinations. 
- **For instance:** TalkBack on Samsung Galaxy 8, or VoiceOver on iPad with magnification.

</details>

CAIA A11ys will review the coverage needed, and self-assign, based on a11y availability.

## Supporting artifacts 

<details><summary>Toggle to view the supporting artifacts</summary>

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/products/ask-va-phase-1-product-outline.md)
- [User flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1708479170047/8ebb517a213e05ebc500087f5661a993a41392e8?wid=1358-1710439493448)
- [Prototype or mockups](https://staging.va.gov/contact-us/ask-va-too/introduction) – Click 'Research study' sign in button
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Dashboard/Research%20plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Dashboard/Conversation%20guide.md)
- [Epics/issue that may be helpful](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87480) – Previous study we worked on with y'all!
- [How to create a new test user account](https://app.zenhub.com/workspaces/ask-va-647a476551689d06655cc815/issues/gh/department-of-veterans-affairs/ask-va/1218)
- [List of test user accounts created for this study](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/ask-va/test-user-accounts.md)
</details>

## Next Steps: Definition of Done

```[tasklist]
### CAIA A11Y Tasks
- [x] **INTAKE TICKET:** @sara-amanda to process intake ticket
- [x] **PILOT CALENDAR HOLDS** - Set **pilot** holds on the A11y [**CAIA Capacity Calendar**](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago)
- [x] **PILOT SESSION MOBILE: SCHEDULE** Pilot session with team. `Conduct around 9/6/2024` `a11y TBD`
- [x] **PILOT SESSION DESKTOP: SCHEDULE** Pilot session with team. `Conduct around 9/6/2024` `a11y TBD`
- [x] **PILOT SESSION MOBILE: CREATE TICKET** See `related tickets task list` in this ticket.
- [x] **PILOT SESSION DESKTOP: CREATE TICKET** See `related tickets task list` in this ticket.
- [x] **CALENDAR HOLDS** Update A11y [CAIA Capacity Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) with holds for user research sessions with AT, when dates become available.
- [x] Receive Source of Truth for Scheduling [Google Sheet](https://docs.google.com/document/d/1vuk0sTW6YV-4mnWf-an5DLlj0tgSD9nnuGa1Y8puVOM/edit?usp=sharing)
- [x] **Requested device breakdown:** Desktop, Mobile and Tablet
- [x] **Divvy user research session support with CAIA A11ys** - based on availability and technology available to support.
- [x] **Updated Holds** Holds updated to specific slots for P1 - P12 on xx/xx/2024 - waiting on official invites (table updated in this ticket as well, based on Roster tab)
- [x] **REMOVE CALENDAR HOLDS** Remove holds on A11y [**CAIA Capacity Calendar**](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) and replace with invitations from project team (or Perigean) Updated holds x/xx/2024
- [x] **P# Shifts:** Updates to the participant numbers cross-referenced with sign-ups and switches between CAIA team members is now complete. 2 slots remain as of 9/13/2024 at 9:24 a.m. EST.
- [x] **Review research folder and assets** `Ready by 9/6/2024`
- [x] **OFFICIAL RAW RESEARCH NOTES** Create [CAIA notes document](https://docs.google.com/document/d/1vuk0sTW6YV-4mnWf-an5DLlj0tgSD9nnuGa1Y8puVOM/edit?usp=sharing)
- [x] **Participate in research**  **Date Span:**`9/17/2024` and `9/24/2024`
- [x] **SYNC:** Connect with CAIA A11ys on research findings (meeting)
- [x] **Share the completed deliverable** with the project team via Slack: `Primary Team Slack` and `Ticket`
- [x] **TEMPLATE SESSION NOTES**: Create [CAIA notes document]for a11ys to use during the sessions
```

```[tasklist]
### VFS TEAM Tasks
- [x] Tag @Naomi Eke in Slack [#sitewide-content-accessibility-ia](https://dsva.slack.com/archives/C01K37HRUAH)
- [x] Your team's researchers and observers read through platform guidance on research with assistive technology users
- [x] Review our [assistive technology pilot guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) and request a pilot, if desired
- [x] Request pilot sessions with CAIA a11ys x2
- [x] Schedule pilot session with CAIA a11ys (Mobile)
- [x] Schedule pilot session with CAIA a11ys (Desktop)
- [x] **PROTOTYPE:** Provide prototype to CAIA A11ys via this ticket in the supporting artifacts section `Available on Sept. 3, 2024 via Staging Link`
- [x] **DATES NEEDED** See `Table` in the ticket above. Please provide date range and slots for AT sessions being planned.
- [x] **ASSISTIVE TECH. INFO NEEDED:** Please provide the AT the users will be utilizing during the research sessions in `Table` in the ticket above.
- [x] **INVITES:** Forward invites from Perigean to CAIA A11ys (Sara)
- [x] **Device Detail:** Desktop, Mobile and Tablet - Device breakdown needed for session prep
- [x] Participant numbers adjusted with Perigean (CAIA to make updates to calendar, table and tracking)
```
```[tasklist]
### Related Tickets
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92024
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92028
```

**Comment by tygindraux** [comment]
Hello! I posted this issue on [Slack](https://dsva.slack.com/archives/C8E985R32/p1725386162991689) today. 

We're hoping to get accessibility support for this study, which will be very similar to our [last study](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87480).

We'd like to run 1-2 pilots with CAIA over the next week. I've added more detail about this in our [research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Dashboard/Research%20plan.md).

Let me know if there are other questions.

**Comment by tygindraux** [comment]
@sara-amanda – I noticed there are pilot tickets created.

Here are some times that work for us:
- Monday, September 9 between 11am - 4pm EST
- Tuesday, September 10 between 11am - 4pm EST
- Wednesday, September 11 between 11am - 4pm EST

Let me know if any of these work for you all 😊

**Comment by sara-amanda** [comment]
## Ask VA Availability
> @sara-amanda – I noticed there are pilot tickets created.
> 
> Here are some times that work for us:
> 
> * Monday, September 9 between 11am - 4pm EST
> * Tuesday, September 10 between 11am - 4pm EST
> * Wednesday, September 11 between 11am - 4pm EST
> 
> Let me know if any of these work for you all 😊

## Scheduled Pilots
- **Mobile Pilot:** 2 -3 p.m. ET on 9/11/2024 with @SarahKay8 ✅ Scheduled
    - Ref. ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/92024

**Comment by tygindraux** [comment]
@sara-amanda – I added you to the study channel, let me know if I should add everyone in CAIA.

Here is the research schedule and it's ready CAIA folks to sign up to support sessions. This is the same format as last time. If you navigate to tab 3, called 'Schedule,' you'll find the sessions. There is a column for an A11y specialist to sign up to each session.

Let me know if you can share with CAIA. Also, I'll create research invites in the calendar tomorrow or Friday which I'll forward to you to add to your capacity calendar.

**Comment by sara-amanda** [comment]
## Research Schedule: Ask VA
Thank you so much @tygindraux ! 🙌 

### Completed
- [x] Received schedule from Ask VA Team
- [x] Linked in ticket to schedule source of truth
- [x] Updated table in ticket with schedule to date
- [x] Shared with CAIA a11ys for sign-ups
- [x] CAIA Internal Research Tracker
- [x] CAIA Internal Notes Doc Created for Sessions
- [x] Requested device breakdown (desktop, tablet and mobile) for sessions

### Next Steps
- Received device breakdown from Ask VA Team
- CAIA a11ys to self-assign 12 sessions on spreadsheet from 9/17-9/24/2024
- Invites to be sent to Sara to add to CAIA Capacity Calendar
- Task list in ticket contains the complete list of next steps

cc: @NaomiPMC @SarahKay8 @coforma-jamie @EvanAtCoforma @mutiaadhocteam @rileyorr

**Comment by tygindraux** [comment]
We don't necessarily need coverage on the screen magnification sessions but SK is welcome to join.

I've updated the invites today based on your changes to the schedule. If there are any issues let me know.

**Comment by SarahKay8** [comment]
![image](https://github.com/user-attachments/assets/f5a12bcd-2a29-458e-8c84-41b41d051fb8)

**Hierarchy and Content Layout**

It seems that the hierarchy of the information could be improved. The question and response should take priority over the time, as they are the most relevant pieces of information. For better scannability, it would be more effective to bold the question and response, placing the time as a secondary line beneath them. Users are generally searching for the content of the interaction, not the timestamp.

**Email Thread Interaction**

Considering how Gmail handles threads, all messages are automatically expanded, with the option to collapse them. Similarly, it might be more intuitive to have all messages initially expanded. This would also enhance searchability, allowing users to search for keywords within the content.

**Accordion Searchability**

When the accordions are collapsed, it prevents users from searching the content inside them. If the accordions were expanded by default, it would make keyword search more effective and user-friendly.

**Message Order and Interface Clarity**

Switching the message order to display the latest message at the top could improve the interface's cleanliness. Expanding only the latest message while keeping previous ones collapsed would create more space and a streamlined look.

**Reply Box Placement**

Would it make more sense to place the reply box immediately after the latest message, or even at the top? Perhaps including a button to focus directly on the reply box could improve navigation and efficiency.

Hey @tygindraux! This is a more in depth feedback summary (options/alternatives) of the accordion page i had concerns about. Could you please pass this to your designers? Thanks in advance 🦋

**Comment by tygindraux** [comment]
Thanks SK! I wrote up some initial replies to these, but looking forward to seeing what we find during research with Veterans, too.

- **Bold the question and response.** This is such a great call out and reordering this hierarchy would better match what users are looking for.
- **Accordions.** We're going to see how it goes in sessions with Veterans re: accordions. I'd like to see if they find they add unnecessary complexity or they're helpful (both pilot sessions had different opinions on this). If we find that all content should be visible then we may choose to have plain content on the page instead of an accordion.
- **Message order.** We've went back and forth on the message order (after initial testing with Veterans) and ultimately decided that having the latest message at the bottom means it's next to the 'send a reply' box so people can reference the latest message while they write their reply. We also found that users expected the page to be in chronological order, starting with initial message and going down from there.
- **Reply box placement.** There's a couple reasons we have this at the bottom of the page. We initially tested with this at the top of the page and users unanimously did not like this. It's distracting when their key task is to read the reply from VA. Also, the reply box is only available sometimes. Some business lines don't allow replies which means sometimes the reply box will be removed from the page (depends on the inquiry type). Having it at the bottom means we can remove it without restructuring the page, which could be more confusing for users who are switching between inquiries that allow replies and those that don't.

I really appreciate you taking the time to write feedback ❤️ I'll share with my team so we can keep this in mind during sessions and see what comes up for Veterans.
